lib/api/redfish-1.0/roles.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var redfish = injector.get('Http.Api.Services.Redfish');
+var controller = injector.get('Http.Services.Swagger').controller;
+var Promise = injector.get('Promise');    // jshint ignore:line
+var _ = injector.get('_'); // jshint ignore:line
+
+var rolesObj = {
```

Comment:
Can you add a comment about how these values are hard coded because they are required values listed in the 1.0 specification?